jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnector.java

```diff
@@ -161,7 +163,18 @@
     private long _idleTimeout = 30000;
     private String _defaultProtocol;
     private ConnectionFactory _defaultConnectionFactory;
+    /* The name used to link up virtual host configuration to named connectors */
     private String _name;
+    /**
+     * The authority used to define the server uri authority fallback for the connection
+     * (see ServletRequest.getServerName(), and ServletRequest.getServerPort()).
+     */
+    private HostPort _serverUriAuthority;
```

Comment:
I think just `_serverAuthority` is a better name.

Reply:
Done.